test/test_tf_frontend_router.py

```diff
@@ -409,20 +413,27 @@ def test_create_fastly_config_backend(self):
         ), cwd=self.workdir).decode('utf-8')
 
         # Then
-        assert re.search(template_to_re("""
-      backend.~{ident}.healthcheck:               ""
-      backend.~{ident}.max_conn:                  "200"
-      backend.~{ident}.name:                      "default backend"
-      backend.~{ident}.port:                      "443"
-      backend.~{ident}.request_condition:         ""
-      backend.~{ident}.shield:                    ""
-      backend.~{ident}.ssl_ca_cert:               ""
-      backend.~{ident}.ssl_cert_hostname:         "foo-foobar.dev.domain.com"
-      backend.~{ident}.ssl_check_cert:            "true"
-      backend.~{ident}.ssl_hostname:              ""
-      backend.~{ident}.ssl_sni_hostname:          ""
-      backend.~{ident}.weight:                    "100"
-        """.strip()), output) # noqa
+        assert """
+      backend.#:                                    "1"
+      backend.~3792877954.address:                  "${var.backend_address}"
+      backend.~3792877954.auto_loadbalance:         "true"
+      backend.~3792877954.between_bytes_timeout:    "30000"
+      backend.~3792877954.connect_timeout:          "5000"
+      backend.~3792877954.error_threshold:          "0"
+      backend.~3792877954.first_byte_timeout:       "60000"
+      backend.~3792877954.healthcheck:              ""
+      backend.~3792877954.max_conn:                 "200"
+      backend.~3792877954.name:                     "default backend"
+      backend.~3792877954.port:                     "443"
+      backend.~3792877954.request_condition:        ""
+      backend.~3792877954.shield:                   ""
+      backend.~3792877954.ssl_ca_cert:              ""
+      backend.~3792877954.ssl_cert_hostname:        "${var.ssl_cert_hostname}"
```

Comment:
You could also leave the `{ident}` and simply skip the `${...}` line?

Reply:
i think you can just use double curlies the was you would with a template.

Reply:
I've had a look at why the `template_to_re` function was preventing it working and it's because it strips out the `{` and `}`s so it just doesn't match. Ideally we could fix that, but I'd just lost patience with it by that point.

Reply:
Like a template, it doesn't match single curlies, you have to double them.